src/blueprints/tv2/action-factories/tv2-split-screen-action-factory.ts

```diff
@@ -104,7 +104,13 @@ export class Tv2SplitScreenActionFactory extends ActionFactory {
       case Tv2ActionSubtype.SPLIT_SCREEN_INSERT_SOURCE_TO_INPUT: {
         return [{
           type: MutateActionType.PIECE,
-          updateActionWithPiece: (action: Action, piece: Piece) => this.updateInsertToInputAction(action, piece),
+          updateActionWithPiece: (action: Action, onAirPiece: Piece | undefined, nextPiece: Piece | undefined): Action => {
+            if (onAirPiece)
+              return this.updateInsertToInputAction(action, onAirPiece)
+            else if (nextPiece)
+              return this.updateInsertToInputAction(action, nextPiece)
+            return action
```

Comment:
```suggestion
            if (onAirPiece) return this.updateInsertToInputAction(action, onAirPiece)
            if (nextPiece) return this.updateInsertToInputAction(action, nextPiece)
            return action
```

no need for "else if"  since you use returns 😄

Reply:
Removed the `else`. Kept the indentation.